ArucoDetectRos-Amir's_Changes.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
+from __future__ import division  #in order to be able to set the (1/30)
+"""
+Framework   : OpenCV Aruco
+Description : Calibration of camera and using that for finding pose of multiple markers
+Status      : Working
+References  :
+    1) https://docs.opencv.org/3.4.0/d5/dae/tutorial_aruco_detection.html
+    2) https://docs.opencv.org/3.4.3/dc/dbb/tutorial_py_calibration.html
+    3) https://docs.opencv.org/3.1.0/d5/dae/tutorial_aruco_detection.html
+"""
+
+from math import floor
+import numpy as np
+import time
+import rospy
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+from sensor_msgs.msg import CameraInfo
+from sensor_msgs.msg import Imu
+import tf
+from std_msgs.msg import UInt32
+from std_msgs.msg import String
+import message_filters
+import tf.transformations as tr
+import numpy as np
+import pickle
+import glob
+import cv2
+import cv2.aruco as aruco
+import csv
+import tf2_ros
+
+calib_path_param = '/home/makeruser/wifi-Project/Aruco_Tracker/images/for_calib/*.jpg'  ########## AMIR -> change these images and location
+aruco_dict_param = aruco.DICT_4X4_250 ########## AMIR -> this work for https://chev.me/arucogen/
```

Comment:
There are TODO comments in here which should be turned into issues/tasks or just removed
```suggestion
calib_path_param = '/home/makeruser/wifi-Project/Aruco_Tracker/images/for_calib/*.jpg'  
aruco_dict_param = aruco.DICT_4X4_250 
```